content/guides/core/automations/automation-events.md

```diff
@@ -0,0 +1,55 @@
+---
+menu:
+  default:
+    identifier: automation-scopes
+    parent: automations
+title: Automation events and scopes
+weight: 2
+---
+{{% pageinfo color="info" %}}
+{{< readfile file="/_includes/enterprise-only.md" >}}
+{{% /pageinfo %}}
+
+An automation can run when a specific event occurs at a given scope, either a registry or a project. This page describes the events that can trigger an automation at each scope.
```

Comment:
Nit: I try to avoid using "run" if it's not in the context of creating a run. (Unfortunate naming of an object).